client/src/components/Menu/dish.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Card, CardHeader, CardMedia, CardContent, Typography } from '@mui/material';
+import AddToCart from './addToCart';
+
+interface DishCardProps {
```

Comment:
Use either type or interface in all your interface declarations, and move them to the interfaces directory.